data/provider/src/main/java/tech/pegasys/teku/api/ValidatorDataProvider.java

```diff
@@ -125,9 +129,15 @@ public boolean isStoreAvailable() {
     if (currentSlot.isGreaterThan(slot)) {
       throw new IllegalArgumentException(CANNOT_PRODUCE_HISTORIC_BLOCK);
     }
+    return validatorApiChannel.createUnsignedBlock(slot, randao, graffiti, false);
```

Comment:
why don't you pass the `isBlinded` param?

Reply:
ah i should really, thats where i started and just a plain old bug